README.md

```diff
@@ -97,11 +69,10 @@ After you're finished please remove all the comments and instructions!
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
 
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Adding Books]**
+- **[Removing Books ]**
+- **[Displaying Books ]**
```

Comment:
- [ ] Again please fix the key feature section of your Readme, currently it looks like the features are of the awesome books project :neutral_face: , while you are developing a todo list where you can add tasks edit them mark them completed etc...